app/app.go

```diff
@@ -244,27 +244,26 @@ func NewApp(
 
 	app.keeper.upgrade = upgrade.NewKeeper(skipUpgradeHeights, keys[upgrade.StoreKey], cdc)
 
-	// no-op handler for "eridani" upgrade
-	app.keeper.upgrade.SetUpgradeHandler("eridani", func(ctx sdk.Context, plan upgrade.Plan) {
-		faucetAddr, _ := sdk.AccAddressFromBech32("akash1dz70tfxxrsh8fned6len7feu3atz7k59zgz77n")
-		personalAcc1, _ := sdk.AccAddressFromBech32("akash1dz70tfxxrsh8fned6len7feu3atz7k59zgz77n")
-		personalAcc2, _ := sdk.AccAddressFromBech32("akash1y4uskp4v6s086pdg6phufqcg82g9xd494kdwmn")
-		valAddr, _ := sdk.ValAddressFromBech32("akashvaloper1753ew9z7cfhu6awyp6ff7qtm9ex30kg3r7zd7j")
+	// no-op handler for "cygni" upgrade
+	app.keeper.upgrade.SetUpgradeHandler("cygni", func(ctx sdk.Context, plan upgrade.Plan) {
+		faucetAddr, _ := sdk.AccAddressFromBech32("akash1czxh6ewhuy00tsv5zu50gz7lz2cxcpufdrarty")
```

Comment:
Why are we hardcoding addresses in app.go?

Reply:
Generally we don't get such situations to mint/burn tokens. This is just an example handler to demonstrate a possibility and I think it's okay to hard-code the addresses here. We can definitely think of a better way to handle it. To keep it simple for everyone to understand the code changes, I use this and did the same in other testnets too.
But we can figureout a better way
https://github.com/regen-network/regen-ledger/commit/bbc9849759e1a75c18e87bb1ed2438709d83023a

Reply:
I'm cool with this given the explanation. Thank you @anilCSE